tests/meltano/core/test_project_files.py

```diff
@@ -182,23 +182,23 @@ def test_load(self, project_files):
                     "extractor": "tap-meltano-yml",
                     "loader": "target-meltano-yml",
                     "transform": "skip",
-                    "start_date": datetime.datetime(2020, 8, 5),  # noqa: WPS432
+                    "start_date": datetime.datetime(2020, 8, 5),  # noqa: DTZ001
```

Comment:
Why not add the tzinfo for these?

Reply:
5213ebd2062b912066854b8a3fc578e73755f008

Reply:
5213ebd2062b912066854b8a3fc578e73755f008